src/core/plugins/postmarketos/api.js

```diff
@@ -0,0 +1,108 @@
+"use strict";
+
+/*
+ * Copyright (C) 2020-2021 UBports Foundation <info@ubports.com>
+ * Copyright (c) 2022 Caleb Connolly <caleb@connolly.tech>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const axios = require("axios");
+
+/** @module postmarketOS */
+
+const baseURL = "https://images.postmarketos.org";
+
+const api = axios.create({ baseURL, timeout: 15000 });
+
+/**
+ * get interfaces from api
+ * @param {String} device device codename
+ * @returns {Promise<Array<String>>} interfaces
+ * @throws {Error} message "unsupported" if 404 not found
+ */
+const getInterfaces = device =>
+  api
+    .get("/bpo/index.json")
+    .then(({ data }) => {
+      let releases = getReleases(device);
```

Comment:
Variables that are not modified should be `const` - also this one is not used, not sure why the linter let it through;